src/app/onboarding/components/OnboardingNotification.vue

```diff
@@ -1,50 +1,48 @@
 <template>
+  <!-- As well as only showing the notification on the homepage -->
+  <!-- We also share the same datasource URL as the mesh listing on the homepage -->
   <DataSource
-    v-if="!route.meta.onboardingProcess"
+    v-if="route.name === 'home'"
     v-slot="{ data }"
-    :src="`/meshes?page=1&size=50`"
+    :src="`/meshes?page=1&size=10`"
   >
     <template
       v-if="data?.items.length === 1 && data.items[0].name === 'default'"
     >
-      <DataSource
-        v-slot="{ data: dataplaneData }"
-        :src="`/meshes/${data.items[0].name}/dataplanes?page=1&size=50&search=`"
+      <template
+        v-if="data.items[0].dataplanes.total === 0"
```

Comment:
Inspect the data from `/mesh-insights` that we already have instead of using an additional DataSource